src/client/java/daos-client/src/main/java/io/daos/dfs/DaosIOException.java

```diff
@@ -21,7 +21,7 @@
  * portions thereof marked with this legend must also reproduce the markings.
  */
 
-package com.intel.daos.client;
+package io.daos.dfs;
```

Comment:
Do you need put classes like this and utilities into a namespace like io.daos.common? Later we will add io.daos.object and the object api may also use these classes.

Reply:
yes. I'll move these classes to common package when we start to work on object API.